Exercise_A/index.html

```diff
@@ -10,6 +10,15 @@
   </head>
   <body>
     <!-- CODE HERE -->
+    <a
+      href="https://mail.google.com/mail/u/1/#inbox?compose=CllgCHrlFplXzTcpCNTBXlKTbDpjGGJdHSkTQwKZZGLJzKzSlvzWSFJXGThKljLRHKmNwWrZvkL"
+      target="_blank"
+      ><img
+        src="/FTWD_Web_Day_02/Exercise_A/gmail.png"
+        alt=""
+        style="width: 100px"
+      />
+    </a>
 
     <script src="" async defer></script>
   </body>
```

Comment:
you need to create about.html file and link your about.html to index.html when clicking the image inside index.html. As for your anchor tag, you need to use mailto to redirect your anchor tag to your email. 
For example,
`<a href="mailto:someone@yoursite.com">Email Us</a> `